index.html

```diff
@@ -0,0 +1,127 @@
+                    <div class="date_tag">
+                    <h3 class="date">Every Saturday with a new Recipe</h3>
+                    <a href="#" class="tag">@myRecipes</a>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </header>
+
+    <main class="main">
+        <div class="main-page" id="main">
+        <h2 class="recipes">Recipes</h2>
+        <div class="container">
+            <div class="recipe-block">
+                <img src="images/rice-bowl.png" alt="image" class="img">
+                <h3 class="category">Main Dishes</h3>
+                <p class="recipe-text">All beefy, veggie</p>
+            </div>
+            <div class="recipe-block">
+                <img src="images/cake.png" alt="image" class="img">
+                <h3 class="category">Desserts</h3>
+                <p class="recipe-text"> Sweet and yummy</p>
+            </div>
+            <div class="recipe-block">
+                <img src="images/pizza-slice.png" alt="image" class="img">
+                <h3 class="category">Snacks</h3>
+                <p class="recipe-text"> Bites and Bites</p>
+            </div>
+            <div class="recipe-block">
+                <img src="images/pizza-slice.png" alt="image" class="img">
+                <h3 class="category">Appetizer</h3>
+                <p class="recipe-text"> First course</p>
+            </div>
+            <div class="recipe-block">
+                <img src="images/pizza-slice.png" alt="image" class="img">
+                <h3 class="category">Drinks</h3>
+                <p class="recipe-text"> Something cool</p>
+            </div>
+        </div>
+        <a href="#">See more recipes</a>
+        <button class="recipe_btn">More Recipes</button>
+        </div>    
+    </main>
+
+    <div class="featured">
+        <div>
+            <h2 class="frecipe">Featured Recipes</h2>
+            <img src="images/line.png" alt="line" class="line">
+            <div class="featurecard">
+            </div>
+        </div>
+        <button class="morebtn"><p>MORE</p><img src="images/pink_arrow.png" alt="arrow" class="arrow"></button>
+    </div>
+    <div class="Mypartners">
+        <h3 class="partners">Partner</h3>
+        <img src="images/line.png" alt="line" class="line1">
+        <div class="brands">
+        <h1 class="brand_names">mozilla</h1>
+        <h1 class="brand_names">Google</h1>
+        <h1 class="brand_names2">NAVER</h1>
+        <h1 class="brand_names1">daumkakao</h1>
+        <h1 class="brand_names1">airbnb</h1>
+        </div>
+    </div>
+    <div class="logo_ri">
+        <img src="images/christmas_2012_new_2857.jpg" alt="logo" class="logofooter">
+        <p class="rights">My Recipes. Some rights reserved</p>
+    </div>
+   
+
```

Comment:
Kindly adjust the footer to match the design as well. The height is considerably larger than that specified in the design. Also, adjust the size and margins of the logo to match.

Your design:
![image](https://user-images.githubusercontent.com/20554583/190858486-58339217-4409-4aa9-8bcf-49f347ba05a6.png)